setup.py

```diff
@@ -65,4 +65,10 @@
             "dev": doc_requirements + test_requirements,
         },
         zip_safe=False,
+        use_scm_version={
+            "write_to": "src/vivarium_cluster_tools/_version.py",
+            "write_to_template": '__version__ = "{version}"\n',
+            "tag_regex": r"^(?P<prefix>v)?(?P<version>[^\+]+)(?P<suffix>.*)?$",
```

Comment:
And this regex is well-tested and/or well-known?

Reply:
It's in the setuptools_scm README as an example.